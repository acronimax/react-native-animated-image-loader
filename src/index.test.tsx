import React from 'react';
import { render } from '@testing-library/react-native';
import AnimatedImgLoader from './index';

describe('react-native-image-loader', () => {
  it('should render correctly', () => {
    const wrapper = render(
      <AnimatedImgLoader
        imageUri={
          'https://images.pexels.com/photos/1366919/pexels-photo-1366919.jpeg'
        }
      />
    );
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toBeDefined();
  });
});
