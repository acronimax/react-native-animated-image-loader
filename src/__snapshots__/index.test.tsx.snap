// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-native-image-loader should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#EEE",
        "borderRadius": 8,
        "height": 444.6666666666667,
        "overflow": "hidden",
        "width": 710,
      },
    ]
  }
>
  <View
    collapsable={false}
    style={
      Object {
        "backgroundColor": "#FFF",
        "bottom": 0,
        "elevation": 24,
        "left": 0,
        "opacity": 0.2,
        "position": "absolute",
        "right": 0,
        "shadowColor": "#FFF",
        "shadowOffset": Object {
          "height": 12,
          "width": 0,
        },
        "shadowOpacity": 0.58,
        "shadowRadius": 16,
        "top": 0,
        "transform": Array [
          Object {
            "translateX": -20,
          },
        ],
        "width": 75,
      }
    }
  />
  <Image
    collapsable={false}
    onLoadEnd={[Function]}
    source={
      Object {
        "uri": "",
      }
    }
    style={
      Object {
        "height": 100,
        "opacity": 0,
        "width": 60,
      }
    }
  />
</View>
`;
